### DISCUSSION TITLE
How can I use RESTAPI `/users/me` for more information about role

### DISCUSSION THREAD

**Discussion by NMTuan** [directus/directus#14175] [Archive]
by default, get `/users/me`,  response `data.role` is an id.
i want more. like `role.name` `role.description` and some fields I added.

**Comment by br41nslug** [comment]
You can add the same filters as other endpoints, so `/users/me?fields[]=*,role.*` should get you the info you want 😄

**Reply by NMTuan** [reply]
wow is cool, thank you.

**Reply by benhaynes** [reply]
@br41nslug — I've never added the array brackets... are those required?

**Reply by br41nslug** [reply]
They are not technically required no. Force of habit left over from when the `[]` was required for browsers/libraries to know that you had an array of data 🤓

**Reply by rijkvanzanten** [reply]
@benhaynes `?fields=a,b` has the same result as `?fields[]=a&fields[]=b`
